packages/core/src/link/Link.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  ComponentType,
+  forwardRef,
+  MouseEvent,
+  ReactElement,
+  useCallback,
+} from "react";
+import cx from "classnames";
+import { IconProps, TearOutIcon } from "@jpmorganchase/uitk-icons";
+import { makePrefixer } from "../utils";
+import { Text, TextProps } from "../text";
+
+import "./Link.css";
+
+const withBaseName = makePrefixer("uitkLink");
+
+/**
+ * Links are a fundamental navigation element. When clicked, they take the user to an entirely different page.
+ *
+ * @example
+ * <LinkExample to="#link">Action</LinkExample>
+ */
+export interface InternalLinkProps extends TextProps<"a"> {
```

Comment:
This naming will probably cause confusion. "Internal" to me sounds like it's a private type that shouldn't be used.